research-app/src/App.vue

```diff
@@ -910,6 +944,12 @@ export default class App extends WWTAwareComponent {
 
   // Background / foreground imagesets
 
+  get curAvailableImagesets() {
+    if (this.wwtAvailableImagesets == null)
+      return [];
+    return this.wwtAvailableImagesets.filter(info => info.type != ImageSetType.panorama);
```

Comment:
The way this filtering should work should be to only select those imagesets whose `type` is equal to `this.wwtRenderType`. The "render type" parameter basically defines the mode that the engine is currently in. Right now, for example, the first few entries in the menu ("Jupiter", "Moon", "Sun", etc.) are planetary maps. Changing the background imageset from one type to another is actually how you tell the engine to switch rendering modes — if you select Jupiter, you'll suddenly switch to the planet-viewer mode, rather than the sky-viewer mode. So showing imagesets of different types isn't a problem per se, but in terms of the UX I think we want to limit the dropdown to only show backgrounds that agree with the current rendering mode.

Also, this should filter out the "progressive HiPS catalogs", which can be identified because they have a FileType containing the text "tsv" (i.e., tab-separated values). I guess that means that the ImageSetInfo struct is going to have to record that piece of information as well — we're going to want to isolate those items for the source-catalog overlay functionality.